selectors/views/threads.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {$ID} from 'mattermost-redux/types/utilities';
+import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
+import {getThreads} from 'mattermost-redux/selectors/entities/threads';
+import {Team} from 'mattermost-redux/types/teams';
+import {UserThread} from 'mattermost-redux/types/threads';
+
+import {GlobalState} from 'types/store';
+import {ViewsState} from 'types/store/views';
+
+export function getSelectedThreadIdInTeam(state: GlobalState) {
+    return state.views.threads.selectedThreadIdInTeam;
+}
+
+export const getSelectedThreadIdInCurrentTeam: (state: GlobalState) => ViewsState['threads']['selectedThreadIdInTeam'][$ID<Team>] = createSelector(
```

Comment:
Do we need to memoize this with `createSelector`? (As in, is there evidence that this selector is both called often and slow to compute?) If not, can we just make this a simple function?

Reply:
(Follow up PR?)